xdsl/irdl.py

```diff
@@ -664,13 +677,18 @@ def verify(self, op: Operation):
         # Verify regions.
         irdl_op_verify_arg_list(op, self, VarIRConstruct.REGION)
 
+        # Verify attributes.
```

Comment:
#random comment
At some point we have to discuss a homogeneous way of writing comments
We should follow some specific style 
https://python.plainenglish.io/how-to-write-numpy-style-docstrings-a092121403ba

e.g. inline comments should start from capital and end with period..something like that...open to ideas

Reply:
While I agree that we should have a comment style, I disagree with the capitalization and period at the end.

I have nothing to back this up other than that it feels much more natural for me to have lowercase comments.

Reply:
(I just wanted to add that this does not extend to docstrings. They should be capitalized and form correct english sentences.)

Reply:
I do not mind, the format, just we can agree o sth homogeneous, nothing urgent at all